### PR TITLE
OSOE-49: Enforce Windows newlines for C# files to avoid false positives with IDE0055 warning.

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OSOE-49](https://lombiq.atlassian.net/browse/OSOE-49)

### CODE REVIEW

**Review by Piedone — APPROVED** [review]

File: Lombiq.Privacy.csproj [hunk 1]
```diff
@@ -57,4 +57,8 @@
     <PackageReference Include="Lombiq.HelpfulLibraries.OrchardCore" Version="4.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\Utilities\Lombiq.Gulp.Extensions\Lombiq.Gulp.Extensions.csproj" />
+  </ItemGroup>
```
Comment (0liver) [inline Lombiq.Privacy.csproj]: Did adding this reference actually help? Can you point me to the run where this fixed the build?

File: .gitattributes [hunk 2]
```diff
@@ -1 +1,5 @@
 * text=auto
+
+# Enforce Windows newlines for C# files to avoid false positives with IDE0055 warning.
+# See https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/106
```
Comment (0liver) [inline .gitattributes]: Let's use proper punctuation here and in the rest of the projects:
```suggestion
# See https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/106.
```

I'll prepare GitHub review suggestions for you, so you can just click through them.

**Review by 0liver — CHANGES_REQUESTED** [review]

File: Lombiq.Privacy.csproj (hunk 1 above)
Comment (sarahelsaig) [reply Lombiq.Privacy.csproj]: I can't for sure, becuase the problem was inconsistent anyway. But I think it's good to have regardless whether it helped or not becuase this project actually uses Gulp. I just moved it to the `<ItemGroup Condition="'$(NuGetBuild)' != 'true'">`.

Comment (0liver) [reply Lombiq.Privacy.csproj]: Fair enough. I agree.

**Review by 0liver — APPROVED** [review]

**Pull request merged**